src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I'm not sure if we need to add this line for such a small change in the file.

Reply:
This line is result of the merge with master (and it was a conflict).